app/services/get-capacity-view.js

```diff
@@ -1,25 +1,85 @@
 const getBreadcrumbs = require('./get-breadcrumbs')
 const getWorkloadReports = require('./data/get-workload-report-views')
+const getCapacityBreakdown = require('./data/get-capacity-breakdown')
 const tableCreator = require('./helpers/table-creator')
 const getOrganisationUnit = require('./helpers/org-unit-finder')
+const organisationConstant = require('../constants/organisation-unit')
+const percentageCalculator = require('./helpers/percentage-calculator')
 
 module.exports = function (id, capacityDateRange, organisationLevel) {
   var organisationalUnitType = getOrganisationUnit('name', organisationLevel)
-  var result = {}
-  var workloadReportsPromise
-
   if (organisationalUnitType === undefined) {
     throw new Error(organisationLevel + ' should be offender-manager, region, team, ldu or hmpps')
   }
 
-  workloadReportsPromise = getWorkloadReports(id, capacityDateRange.capacityFromDate.toISOString(), capacityDateRange.capacityToDate.toISOString(), organisationLevel)
-
+  var result = {}
   result.breadcrumbs = getBreadcrumbs(id, organisationLevel)
 
-  return workloadReportsPromise.then(function (results) {
+  return getWorkloadReports(id, capacityDateRange.capacityFromDate.toISOString(), capacityDateRange.capacityToDate.toISOString(), organisationLevel)
+  .then(function (results) {
+    result.capacityBreakdown = []
     result.capacityTable = tableCreator.createCapacityTable(id, organisationalUnitType, capacityDateRange, results)
     result.title = result.breadcrumbs[0].title
     result.subTitle = organisationalUnitType.displayText
+
+    if (organisationalUnitType !== organisationConstant.OFFENDER_MANAGER) {
+      return getCapacityBreakdown(id, organisationLevel)
```

Comment:
The `getCapacity` function takes into account the start and end dates selected. I'm assuming the breakdown should also do this?

Reply:
The capacity breakdown shows the capacity in the current workload, while the getCapcity get the history for the graph.  The capacity breakdown expands on the current capacity column in the overview

Reply:
That makes sense. Thanks for clarifying.